src/opnsense/www/themes/opnsense-dark/assets/stylesheets/config/colors.scss

```diff
@@ -12,4 +12,6 @@ $colors: (
   border-default: #E5E5E5,
   dashboard-widget-error: #721c24,
   border-highlight: #AAA,
+  count-badge-background: #31708f,
```

Comment:
I may have led you on the wrong path here, but since these colors are pretty specific to the bootgrid css files, it's better to define these colors in the `opnsense-boogrid.scss` files:

Instead of:

`background-color: map-get($colors, count-badge-background);`

do:

`background-color: #31708f`